docs/reference/modules/discovery/azure.asciidoc

```diff
@@ -1,6 +1,4 @@
 [[modules-discovery-azure]]
 === Azure Discovery
 
-Azure discovery allows to use the Azure APIs to perform automatic discovery (similar to multicast).
-Please check the https://github.com/elasticsearch/elasticsearch-cloud-azure[plugin website]
-to find the full documentation.
+Azure discovery is available as a plugin. See {plugins}/discovery-azure.html[discovery-azure] for more information.
```

Comment:
Well I tried to have the same structure as for ec2: https://github.com/elastic/elasticsearch/blob/master/docs/reference/modules/discovery/ec2.asciidoc

BTW with your comment I found a link error in https://github.com/elastic/elasticsearch/blob/master/docs/reference/modules/discovery/gce.asciidoc. I think I'll open a doc PR to fix that and ec2 doc.

But for now, I will apply your comment to azure doc.